.github/workflows/benchmark_deep_frontend.yml

```diff
@@ -0,0 +1,96 @@
+name: Benchmark GTSFM on select datasets using deep front-end
+on: [pull_request]
+jobs:
+  benchmark-deep:
+    name: Run GTSFM
```

Comment:
thanks for the PR, @ayushbaid. Could we prevent any copy-paste by looping over the config name in the workflow matrix?

Reply:
i think this job is already 3 benchmark jobs right? i would probably prefer no copy-paste to make maintenance easier